### ISSUE TITLE
Feature Request: Add graphs to stack and service pages

### ISSUE THREAD

**Issue by AlexGustafsson** [swarmpit/swarmpit#512]
Currently, the dashboard (`swarmpit.example.com/`) and task page (`swarmpit.example.com/tasks/<id>`) feature graphs covering CPU, memory usage etc.

It would be really nice if similar graphs could be used on the stack (`swarmpit.example.com/stacks/<id>`) and service (`swarmpit.example.com/services/<id>`) pages as well.

To be entirely clear, I've added a mockup:

![mockup](https://user-images.githubusercontent.com/14974112/82728629-db637080-9cf1-11ea-9faa-d7b7c215598e.png)

Basically, I would want the same graphs as on the dashboard but for each stack and service.
